wifiphisher/pywifiphisher.py

```diff
@@ -187,7 +187,8 @@ def shutdown(template=None, network_manager=None):
         template.remove_extra_files()
 
     print '[' + R + '!' + W + '] Closing'
-    sys.exit(0)
+    for c in phishinghttp.creds:
+        sys.exit('[' + R + '+' + W + '] Captured credentials:' + R + c)
```

Comment:
Yes, you're right, same termination is called when you ctrl + c, yet the -qS seems to clear the credentials every time (except with this "patch").

I have a feeling that it's got something to do with the logging.

I'm thinking about creating a single merged log stream that wifiphisher will tail because 3 different tailing log files and lines inbetween them doesn't seem like the best option to me.

What do you think might cause the issue?
Have you tested this on your system?